docs/references.bib

```diff
@@ -1,689 +1,749 @@
-      year = {2018},
-      eprint = {1802.01795},
-      archivePrefix={arXiv},
-      primaryClass={math.LO}
-}
-
-@article {MR317916,
-    AUTHOR = {Davis, Martin},
-     TITLE = {Hilbert's tenth problem is unsolvable},
-   JOURNAL = {Amer. Math. Monthly},
-  FJOURNAL = {American Mathematical Monthly},
-    VOLUME = {80},
-      YEAR = {1973},
-     PAGES = {233--269},
-      ISSN = {0002-9890},
-   MRCLASS = {02G05 (10B99 10N05)},
-  MRNUMBER = {317916},
-MRREVIEWER = {R. L. Goodstein},
-       DOI = {10.2307/2318447},
-       URL = {https://doi.org/10.2307/2318447},
-}
-
-@article {MR32592,
-    AUTHOR = {Motzkin, Th.},
-     TITLE = {The {E}uclidean algorithm},
-   JOURNAL = {Bull. Amer. Math. Soc.},
-  FJOURNAL = {Bulletin of the American Mathematical Society},
-    VOLUME = {55},
-      YEAR = {1949},
-     PAGES = {1142--1146},
-      ISSN = {0002-9904},
-   MRCLASS = {09.1X},
-  MRNUMBER = {32592},
-MRREVIEWER = {B. N. Moyls},
-       DOI = {10.1090/S0002-9904-1949-09344-8},
-       URL = {https://doi.org/10.1090/S0002-9904-1949-09344-8},
-}
-
-@incollection {MR541021,
-    AUTHOR = {Nagata, Masayoshi},
-     TITLE = {On {E}uclid algorithm},
- BOOKTITLE = {C. {P}. {R}amanujam---a tribute},
-    SERIES = {Tata Inst. Fund. Res. Studies in Math.},
-    VOLUME = {8},
-     PAGES = {175--186},
- PUBLISHER = {Springer, Berlin-New York},
-      YEAR = {1978},
-   MRCLASS = {13F07},
-  MRNUMBER = {541021},
-MRREVIEWER = {Daniel Lazard},
-}
-
-@article {MR399081,
-    AUTHOR = {Hiblot, Jean-Jacques},
-     TITLE = {Des anneaux euclidiens dont le plus petit algorithme n'est pas
-              \`a valeurs finies},
-   JOURNAL = {C. R. Acad. Sci. Paris S\'{e}r. A-B},
-  FJOURNAL = {Comptes Rendus Hebdomadaires des S\'{e}ances de l'Acad\'{e}mie des
-              Sciences. S\'{e}ries A et B},
-    VOLUME = {281},
-      YEAR = {1975},
-    NUMBER = {12},
-     PAGES = {Ai, A411--A414},
-      ISSN = {0151-0509},
-   MRCLASS = {13F15 (12A20)},
-  MRNUMBER = {399081},
-MRREVIEWER = {N. Sankaran},
-}
-
-@book{calugareanu,
-      author = {C\v{a}lug\v{a}reanu, Grigore},
-      year = {2000},
-      month = {01},
-      pages = {},
-      title = {Lattice Concepts of Module Theory},
-      doi = {10.1007/978-94-015-9588-9}
-}
-
-@Article{Stone1979,
-  author     = {Stone, A. H.},
-  journal    = {General Topology Appl.},
-  title      = {Inverse limits of compact spaces},
-  year       = {1979},
-  issn       = {0016-660X},
-  number     = {2},
-  pages      = {203--211},
-  volume     = {10},
-  doi        = {10.1016/0016-660x(79)90008-4},
-  fjournal   = {General Topology and its Applications},
-  mrclass    = {54B25},
-  mrnumber   = {527845},
-  mrreviewer = {J. Segal}
+# To normalize:
+# bibtool --preserve.key.case=on --preserve.keys=on -s -i docs/references.bib -o docs/references.bib
+
+
+@Article{	  ahrens2017,
+  author	= {Benedikt Ahrens and Peter LeFanu Lumsdaine},
+  year		= {2019},
+  title		= {Displayed Categories},
```

Comment:
Using tabs to indent renders pretty poorly on github

Reply:
I noticed this in [a community website PR](https://github.com/leanprover-community/leanprover-community.github.io/pull/161), but this seems to be an issue only with the diff view. It looks fine if you "View file": https://github.com/leanprover-community/mathlib/blob/f60b0753eb820c2f9be85bc26e3af5fb65ab83af/docs/references.bib

Reply:
Looks like there's an option in bibtool to use spaces instead of tabs, so I switched to that. Unfortunately running bibtool deletes the comment at the top...

Reply:
Maybe we should start compiling a new doc page for citations / .bib stuff on the website?